src/main/fc/rc.c

```diff
@@ -496,134 +496,182 @@ NOINLINE void initAveraging(uint16_t feedforwardAveraging)
+    // Greater feedforward_jitter_factor values widen the attenuation range, and increase the suppression at center
+    // Stick input is the average of the previous two absolute rcCommandDelta values
+    // Output is jitterAttenuator, a value 0-1.0 that is a simple multiplier of the final feedforward value
+    // For the CLI user setting of feedforward_jitter_factor:
+    // User setting of 0 returns feedforwardJitterFactorInv = 1.0 (and disables the function)
+    // User setting of 1 returns feedforwardJitterFactorInv = 0.5
+    // User setting of 9 returns feedforwardJitterFactorInv = 0.1
+    // rcCommandDelta has 500 unit values either side of center stick position
+    // For a 250Hz link, a one second stick sweep center->max returns rcCommandDelta around 2.0
+    // For a user jitter reduction setting of 2, the jitterAttenuator value ranges linearly
+    // from 0.33 when rcCommandDelta is close to zero, up to 1.0 for rcCommandDelta of 2.0 or more
+    // For a user jitter reduction setting of 9, the jitterAttenuator value ranges linearly
+    // from 0.1 when rcCommandDelta is close to zero, up to 1.0 for rcCommandDelta is 9.0 or more
+    // note that the jitter reduction multiplies the final smoothed value of feedforward
+    // allowing residual smooth feedforward offsets even if the sticks are not moving
+    // this is an improvement on the previous version which 'chopped' FF to zero when sticks stopped moving
+    float jitterAttenuator = ((rcCommandDeltaAbs + prevRcCommandDeltaAbs[axis]) * 0.5f + 1.0f) * pid->feedforwardJitterFactorInv;
+    jitterAttenuator = MIN(jitterAttenuator, 1.0f);
+    prevRcCommandDeltaAbs[axis] = rcCommandDeltaAbs;
 
     // smooth the setpointSpeed value
     setpointSpeed = prevSetpointSpeed[axis] + pid->feedforwardSmoothFactor * (setpointSpeed - prevSetpointSpeed[axis]);
 
-    // calculate acceleration and attenuate
-    setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * rxRate * 0.01f;
+    // calculate setpointAcceleration from smoothed setpoint speed
+    setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]);
     prevSetpointSpeed[axis] = setpointSpeed;
 
-    // smooth the acceleration element (effectively a second order filter) and apply jitter reduction
+    // smooth the acceleration element (effectively a second order filter since incoming setpoint was already smoothed)
     setpointAcceleration = prevAcceleration[axis] + pid->feedforwardSmoothFactor * (setpointAcceleration - prevAcceleration[axis]);
-    prevAcceleration[axis] = setpointAcceleration * zeroTheAcceleration;
-    setpointAcceleration = setpointAcceleration * pid->feedforwardBoostFactor * jitterAttenuator * zeroTheAcceleration;
+    prevAcceleration[axis] = setpointAcceleration;
+
+    // apply gain factor to acceleration
+    setpointAcceleration *= rxRate * pid->feedforwardBoostFactor;
+
+    feedforward = setpointSpeed;
+
+    if (axis < FD_YAW) {
```

Comment:
More explicit 
```suggestion
    if (axis == FD_ROLL || axis == FD_PITCH) {
```

Reply:
Not sure, I think it is clear as is, and only involves one comparison.
We know there are only three axes and that yaw is last
Other alternatives are `axis != FD_YAW`?
Or perhaps reverse the condition to handle yaw first, ie `axis == FD_YAW` code then `else...` for pitch and roll.
What do you think is truly best here?

Reply:
@ctzsnooze : gcc is smart enough to generate single comparison in both cases (handling of axis < 0 is different).

`axis < FD_YAW` (or `axis <= FD_PITCH`) is quite idiomatic in BF, but problematic. It takes outsiders time to learn order of axes and it looks suspicious - why is so complicated code used for such a simple thing.

Reply:
Thanks @haslinghuis for your suggestion, and @ledvinap for your advice, have used the above suggestion